app/models/manageiq/providers/embedded_automation_manager/authentication.rb

```diff
@@ -20,8 +20,12 @@ def self.display_name(number = 1)
 
   include ManageIQ::Providers::EmbeddedAutomationManager::CrudCommon
 
-  def self.params_to_attributes(_params)
-    raise NotImplementedError, "must be implemented in a subclass"
+  def self.params_to_attributes(params)
+    allowed_params     = self::API_ATTRIBUTES.pluck(:id) + %w[name type options]
```

Comment:
No because `API_ATTRIBUTES` is used by ui-classic to build dynamic dialogs (https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/ansible_credential_helper/textual_summary.rb#L15) so if we add them there they will show up on the UI which we don't want.